package.json

```diff
@@ -4,26 +4,29 @@
   "description": "A Social platform for the creative at heart",
   "main": "index.js",
   "scripts": {
-    "test": "cross-env NODE_ENV=development nyc mocha --require @babel/register ./src/__tests__/*.js",
+    "test": "cross-env NODE_ENV=development nyc mocha --require @babel/register ./src/__tests__/*.js --timeout 45000 --exit",
     "coverage": "nyc report --reporter=text",
     "coveralls": "nyc report --reporter=text-lcov | coveralls",
     "lint": "eslint \"**/*.js\" --ignore-pattern node_modules/ --fix",
     "build": "babel ./src -d dist",
     "start": "node dist/index.js",
-    "start:dev": "nodemon --exec babel-node -- src/index.js",
+    "sync": "DEBUG=dev babel-node ./src/database/reSync",
+    "start:dev": "npm run sync && nodemon --exec babel-node -- src/index.js",
```

Comment:
Please remove the `sync` from the `start:dev` script. Include in the PR description that the user should run `npm run sync` before running `start:dev`. Not everyone will want to run sync every time they start the application.

Reply:
I get this error when I run `sync`.
<img width="1440" alt="Screenshot 2019-08-28 at 10 51 34 AM" src="https://user-images.githubusercontent.com/27569442/63845423-e0250f00-c981-11e9-9d41-13b61cfb046a.png">

Reply:
I requires YOUR_EMAIL and SOME_PASSWORD field in your .env to create an account for you to test application. This is because there is currently no database record.